cfg/config.go

```diff
@@ -3,11 +3,84 @@
 // that will be retrieved from  .env or ./cfg/config.yml
 package cfg
 
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+)
+
+// Load configs from a env file & sets them in environment variables .
+func LoadEnvVar() error {
+	f, err := os.Open(".env")
+
+	if err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	defer func() {
+		err := f.Close()
+		if err != nil {
+			log.Printf("%s", err)
+		}
+	}()
+
+	lines := make([]string, 0)
+	scanner := bufio.NewScanner(f)
+
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+
+	if err := scanner.Err(); err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	for _, l := range lines {
+		pair := strings.Split(l, "=")
```

Comment:
Are you sure that pairs always have at least 2 elements?
What happen if there will be only 1 element?
We have to handle such a case.

Reply:
Done.